lib/fog/google/models/storage_json/files.rb

```diff
@@ -0,0 +1,93 @@
+require "fog/core/collection"
+require "fog/google/models/storage_json/file"
+
+module Fog
+  module Storage
+    class GoogleJSON
+      class Files < Fog::Collection
+        extend Fog::Deprecation
+        deprecate :get_url, :get_https_url
+
+        attribute :common_prefixes, :aliases => "CommonPrefixes"
+        attribute :delimiter,       :aliases => "Delimiter"
+        attribute :directory
+        # attribute :is_truncated,    :aliases => "IsTruncated"
```

Comment:
I'm not sure, but seems like it is from the Amazon-compatible API. If so - maybe it should be removed? If not - at least put a TODO